features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.infer.json

```
@@ -308,7 +308,8 @@
       "scim2.return_conflict_on_claim_uniqueness_violation": false,
       "authentication.authenticator.office365.parameters.UseOrgSpecificCommonAuthURL": false,
       "scim2.throw_userstore_exception_on_user_creation_error": false,
-      "authentication.local_authenticators.hide_user_existence": false
+      "authentication.local_authenticators.hide_user_existence": false,
+      "saml.slo.front_channel.post_binding.logout_request_signature_with_tenant_cert": true
```

```suggestion
      "saml.slo.front_channel.post_binding.logout_request_signature_with_tenant_cert": false
```
This should be set to false to keep the existing behaviour for 7.0 and 7.1 product?